ja/core-libraries/internationalization-and-localization.rst

```diff
@@ -455,7 +455,7 @@ CakePHP が利用しているものと同じやり方を使い続けることも
 
     I18n::config('default', function ($domain, $locale) {
         $locale = Locale::parseLocale($locale);
-        $language = $locale['language'];
+        $lang = $locale['language'];
```

Comment:
follow it

- corrected variable mismatch: https://github.com/cakephp/docs/pull/6400